problem_builder/mcq.py

```diff
@@ -123,7 +123,11 @@ def author_edit_view(self, context):
         fragment = Fragment(u"<p>{}</p>".format(self.question))
         self.render_children(context, fragment, can_reorder=True, can_add=False)
         fragment.add_content(loader.render_template('templates/html/questionnaire_add_buttons.html', {}))
+        fragment.add_css_url(self.runtime.local_resource_url(self, 'public/css/problem-builder.css'))
         fragment.add_css_url(self.runtime.local_resource_url(self, 'public/css/questionnaire-edit.css'))
+        fragment.add_javascript_url(self.runtime.local_resource_url(self, 'public/js/problem_builder.js'))
+        fragment.add_javascript_url(self.runtime.local_resource_url(self, 'public/js/questionnaire_edit.js'))
+        fragment.initialize_js('QuestionnaireEdit')
```

Comment:
Do you think there's a way to modify the `QuestionnaireAbstractBlock.author_edit_view` so that these subclasses can just call the superclass method with minor modifications? Right now, there is a lot of repeated code in the various `author_edit_view` implementations.